### ISSUE TITLE
Remove-PSFConfig fails for a simple test

### ISSUE THREAD

**Issue by Splaxi** [PowershellFrameworkCollective/psframework#373]
Expected behavior:

This should delete the newly created configuration.

```
Set-PSFConfig -FullName "test.test.test" -Value "Test" -AllowDelete
Remove-PSFConfig -FullName "test.test.test"
```

But it fails with an internal error:

```
Write-PSFMessage : Value cannot be null.
Parameter name: args
At C:\Users\motz\OneDrive - Essence Solutions P S\Documents\WindowsPowerShell\Modules\PSFramework\1.0.19\PSFramework.ps
m1:2250 char:21
+ ... ($result) { Write-PSFMessage -Level InternalComment -String Configura ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-PSFMessage], ArgumentNullException
    + FullyQualifiedErrorId : System.ArgumentNullException,PSFramework.Commands.WritePSFMessageCommand
```

Digging into the module, it seems like inside the `Remove-PSFConfig function` there is an issue with this line:

https://github.com/PowershellFrameworkCollective/psframework/blob/dc8c54b92d43c5a0b98c580f9f95f1a4d204020a/PSFramework/functions/configuration/Remove-PSFConfig.ps1#L98

More specifically with the `-StringValues $item.FullName` part.

Changing it to -StringValues @($item.FullName) seems to fix the issue.

I'll create a PR with the suggested fix and then you can have a look at it.

**Comment by FriedrichWeinmann** [comment]
Bug found and fixed, will be resolved in the next release

**Comment by FriedrichWeinmann** [comment]
Resolved since 1.0.33
